### PR TITLE
New version: GIMP.GIMP.Nightly version 3.0.0-RC1

### LINKED ISSUE #189011: [Update Request]: GIMP
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

GIMP.GIMP

### Package Version

3.0.0-RC1

### Please describe the changes you would like to see

First release candidate for the upcoming 3.0 version.

https://download.gimp.org/gimp/v3.0/windows/
https://download.gimp.org/gimp/v3.0/windows/gimp-3.0.0-RC1-setup.exe

### PR DESCRIPTION
**Pull request by Dragon1573**
## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Resolve #189011

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

## Manual validation

Image
Image

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/189037)

### CODE REVIEW

**Review by ItzLevvie — APPROVED** [review]

File: manifests/g/GIMP/GIMP/Nightly/3.0.0-RC1/GIMP.GIMP.Nightly.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Created with komac v2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: GIMP.GIMP.Nightly
+PackageVersion: 3.0.0-RC1
+InstallerLocale: en-US
```
Comment (Dragon1573) [inline manifests/g/GIMP/GIMP/Nightly/3.0.0-RC1/GIMP.GIMP.Nightly.installer.yaml:6]:
```suggestion
InstallerLocale: en-US
MinimumOSVersion: 10.0.0.0
```

File: manifests/g/GIMP/GIMP/Nightly/3.0.0-RC1/GIMP.GIMP.Nightly.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+Platform:
+- Windows.Desktop
+InstallerType: inno
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ProductCode: GIMP-3_is1
```
Comment (Dragon1573) [inline manifests/g/GIMP/GIMP/Nightly/3.0.0-RC1/GIMP.GIMP.Nightly.installer.yaml:15]:
```suggestion
```

File: manifests/g/GIMP/GIMP/Nightly/3.0.0-RC1/GIMP.GIMP.Nightly.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+ReleaseDate: 2024-11-05
+AppsAndFeaturesEntries:
+- Publisher: The GIMP Team
+  DisplayVersion: 3.0.0.0
+  ProductCode: GIMP-3_is1
```
Comment (Dragon1573) [inline manifests/g/GIMP/GIMP/Nightly/3.0.0-RC1/GIMP.GIMP.Nightly.installer.yaml:20]:
```suggestion
```

**Review by Dragon1573 — COMMENTED** [review]
Suggestions from comments, for manifests consistency.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**